pkgs/tools/misc/tewisay/default.nix

```diff
@@ -1,35 +1,40 @@
-{ lib, buildGoPackage, fetchFromGitHub, makeWrapper }:
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, unstableGitUpdater
+}:
 
-buildGoPackage rec {
-  pname = "tewisay-unstable";
-  version = "2017-04-14";
-
-  goPackagePath = "github.com/lucy/tewisay";
+buildGoModule rec {
+  pname = "tewisay";
+  version = "unstable-2022-11-04";
 
+  # lucy deleted the old repo, this is a fork/mirror
   src = fetchFromGitHub {
-    owner = "lucy";
+    owner = "raymond-w-ko";
     repo = "tewisay";
-    rev = "e3fc38737cedb79d93b8cee07207c6c86db4e488";
-    sha256 = "1na3xi4z90v8qydcvd3454ia9jg7qhinciy6kvgyz61q837cw5dk";
+    rev = "caa5b0131dda868f656716d2107f02d04d1048d4";
+    hash = "sha256-E492d8P/Bek9xZlJP+k9xvIJEFtA1YrIB/pogvz3wM4=";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
+  vendorHash = "sha256-WcpRJ31kqWA255zfjuWDj0honJgSGdm4ONx2yOKk7/g=";
 
-  goDeps = ./deps.nix;
+  # Currently hard-coded, will be fixed by developer
```

Comment:
Where was this communicated?

Reply:
https://github.com/raymond-w-ko/tewisay/blob/caa5b0131dda868f656716d2107f02d04d1048d4/main.go#L138
I've been making a habit of searching for ``/usr`` and other hard-codes recently.

Reply:
Would be nice to link those  in the comment.